.github/workflows/test.yml

```diff
@@ -1,6 +1,11 @@
 name: Build & Test
 
-on: [push]
+on:
+  push:
+    branches: [master, main, develop]
+  pull_request:
+    branches: [master, main, develop]
+  workflow_dispatch:
 
```

Comment:
The ci for test will now run as well, so why add this?

Reply:
The push only for us who have access to the repo. It does not trigger by PRs for contribution.